src/schemas/21-4142/schema.js

```diff
@@ -0,0 +1,67 @@
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+import _ from "lodash";
+
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'AUTHORIZATION TO DISCLOSE INFORMATION TO THE DEPARTMENT OF VETERANS AFFAIRS (21-4142/21-4142a)',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'address',
+    'phone',
+    'dateRange'
+  ]),
+  anyOf: [
+    {
+      "required" : ["vaFileNumber"]
+    },
+    {
+      "required" : ["veteranSocialSecurityNumber"]
+    }
+  ],
+  properties: {
+    email: {
+      type: 'string',
+      format: 'email'
+    },
+    phone: {
+      $ref: '#/definitions/phone'
+    },
+    limitedConsent: {
+      type: 'string'
+    },
+    providerFacility: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          providerFacilityName: {
+            type: 'string'
+          },
+          treatmentDateRange: {
+            $ref: '#/definitions/dateRange'
+          },
+          providerFacilityAddress: {
+            $ref: '#/definitions/address'
+          }
+        }
+      }
+    }
+  },
+  required: ['privacyAgreementAccepted', 'veteranFullName']
+};
+
+[
+  ['privacyAgreementAccepted'],
+  ['fullName', 'veteranFullName'],
+  ['ssn', 'veteranSocialSecurityNumber'],
+  ['vaFileNumber'],
+  ['date', 'veteranDateOfBirth'],
+  ['address', 'veteranAddress']
```

Comment:
this form is going to be converted into PDF and submitted to the central mail API right? you should use the special `centralMailAddress` and `centralMailVaFile` definitions. when making a pdf submission request the API also requires the file number and zip code to be submitted as part of the `metadata` and they have to be formatted in a certain way. the normal address definition can be used for `providerFacilityAddress` since they are probably expecting the zipcode from `veteranAddress`.